.eslintrc

```diff
@@ -91,6 +86,7 @@
     "no-unused-expressions": 0,
     "no-unused-vars": 2,
     "no-use-before-define": 0,
+    "react/display-name": 0,
```

Comment:
Instead of disabling this project-wide, you can always add a line like `// eslint-disable-next-line react/display-name` above the specific use case where you want to disable this rule.

Reply:
ah ok that's probably better. The problem with display names mainly occurred in the `field-definitions.js` file on the `react-port` branch, so maybe I'll just add a comment to cancel display name checks for that file alone.